### ISSUE TITLE
New crafting recipes

### ISSUE THREAD

**Issue by scribblemaniac** [TheValarProject/AwakenDreamsClient#61]
There are plenty of items and blocks in the mod, yet almost none of them have crafting recipes right now. I would like to see more many more crafting recipes in v0.5. We will use this issue to track proposed recipes. Thanks to @MartyParty21's new crafting system, we also have to figure out what recipes should be specific to certain crafting tables.

Similar issue for crafting furnaces: #63

**Comment by MartyParty21** [comment]
This is a list of weapons we have so far, divided into categories. This list shows what weapons should be craftable at various crafting table depending on the race. Legendary weapons however should not be craftable.

This list can be used for #64 to determine how hard it should be to repair weapons.

- Legendary weapons
  - sting
  - broken narsil
  - boromirs sword
  - sharkus shortsword
  - anduril
  - aicanar
  - balins sword
  - bifurs spear
  - bofurs spear
  - bomburs spoon
  - doris sword
  - dwalin axe
  - filis knife
  - filis sword
  - kilis sword
  - noris club
  - oins staff 
  - orcrist
  - thorins sword t
  - hrains warhammer 
  - thrors warhammer 
  - aragorns elf knife 
  - eowyns sword 
  - gandalf the greys staff 
  - gandalf the whites staff 
  - gimlis two-headed axe 
  - gimlis longaxe 
  - glamdring 
  - gloins axe 
  - glorfindels sword 
  - guthwine 
  - hadhafang 
  - legolas sword 
  - radagasts staff 
  - lurtz sword 
  - saurons mace 
  - sarumans staff 
  - yaznegs axe
- Normal
  - Orcs and Uruks and other Evil factions
  - Uruk sword
  - Orc swords 1, 2, 3
  - goblin sword
  - morgul sword
  - berserker sword
  - dol guldur spiked mace 
  - fell-wargrider sword 
  - haradrim snake dagger 
  - corsair eket 
  - north-goblin mace 
  - north-goblin sword 
  - orc captain sword 
  - orc dagger 
  - orc halbard 
  - uruk siege-trooper dagger 
  - morannon dagger 
  - orc spear
- Elves
  - rivendell sword
  - elven long sword
  - mace of glory ?
- Men
  - Rohan
    - rohirrim axe
    - rohirrim spear 
  - Gondor
    - sword of westernesse 
    - gondorian noble sword 
    - gondorian spear 
- Hobbits
  - hobbit sword
  - hobbit axe
  - hobbit hammer
  - hobbit bow
- Dwarves
  - dwarven axe moria
  - dwarven lord knife 
  - golden war hammer of erebor 
- Neutral
  - battle pickaxe
  - pike club
  - shirriff club
  - two handed sword
  - throwing stone
  - iron mace
  - bull-head mace

**Comment by scribblemaniac** [comment]
@MartyParty21 I totally agree with the system you've proposed, I was thinking of something similar myself. Good work developing that list!
